parent/pom.xml

```diff
@@ -1395,42 +1395,32 @@
             </build>
         </profile>
         <profile>
+            <id>Performance</id>
+            <properties>
+                <includedTestGroups>Performance</includedTestGroups>
+                <excludedTestGroups>Live,WIP,Broken</excludedTestGroups>
+            </properties>
+        </profile>
+        <profile>
             <id>Acceptance</id>
             <properties>
                 <includedTestGroups>Acceptance</includedTestGroups>
                 <excludedTestGroups>Integration,Live,WIP,Broken</excludedTestGroups>
```

Comment:
Let's take this to the dev@brooklyn mailing list. I'll email something out today.